content_schemas/examples/organisation/frontend/number_10.json

```diff
@@ -1574,10 +1574,7 @@
             "title": "",
             "href": "",
             "summary": "The Number 10 media blog provides an opportunity to share the government’s position on a wide range of issues directly to the public.",
-            "image": {
-              "url": "https://assets.publishing.service.gov.uk/government/uploads/system/uploads/promotional_feature_item/image/67/No10-door-image.png",
-              "alt_text": ""
-            },
+            "youtube_video_id": "fFmDQn9Lbl4",
```

Comment:
Nice! I imagine F&V can use this example schema to develop against (well if such facilities still exist - they did once)

Reply:
Yup absolutely 👍